.github/workflows/linux.yml

```diff
@@ -499,6 +499,79 @@ jobs:
       runner: ${{ github.event_name == 'schedule' && 'ubuntu-20.04-16-cores' || 'ubuntu-20.04-8-cores' }}
       event: ${{ github.event_name }}
 
+#  E2E_TensorFlow_Models_Tests:
+#    name: E2E TensorFlow Hub Models tests
+#    needs: Build
+#    defaults:
+#      run:
+#        shell: bash
+#    runs-on: ${{ github.event_name == 'schedule' && 'ubuntu-20.04-16-cores' || 'ubuntu-20.04-8-cores'}}
+#    env:
+#      INSTALL_DIR: ${{ github.workspace }}/install
+#      INSTALL_TEST_DIR: ${{ github.workspace }}/install/tests
```

Comment:
let us remove it for now and add when you will activate it in GHA. Now I afraid that this is not working

Reply:
Done